src/extension.ts

```diff
@@ -89,8 +142,98 @@ export async function activate(context: vscode.ExtensionContext) {
 		let envVars = config.get<NodeJS.Dict<string>>('toolsEnvVars', {});
 		envVars.GOROOT = goroot ? goroot: undefined;
 		envVars.GOFLAGS = buildTags ? "-tags="+(buildTags.split(' ').join(',')) : undefined;
+		envVars.TARGET = target;
+		envVars.TINYGOROOT = tinygoroot;
+		envVars.GdbPath = path.basename(gdbPath);
+		envVars.OpenocdInterface = openocdInterface;
+		envVars.OpenocdTarget = openocdTarget;
+		envVars.TmpDir = os.tmpdir();
```

Comment:
Why are you storing all of these in .vscode/settings.json? I don't see how they are necessary for the Go extension.